test/integration/data.json

```diff
@@ -450,14 +450,14 @@
   },
   {
     "file":"grappa_3d/ref_grappa2x1.h5",
-    "md5":"309ba1d310dc0dbbe0a06cf4e0c8444c"
+    "md5": "1e7fa308d1747928007fc55b519344fc"
```

Comment:
Not totally clear why the hashes for these files are changes on this PR?

Reply:
Honestly I have no idea. Someone must have changed the underlying files.